arena_xpos: 60.0
arena_xneg: -1.0
arena_ypos: 15.0
arena_yneg: -15.0
